next/platform/ios/ios.cmake

```diff
@@ -112,6 +112,17 @@ if(MBGL_IOS_RENDER_TEST)
         ${MBGL_ROOT}/render-test/ios/ViewController.m
         ${MBGL_ROOT}/render-test/ios/iosTestRunner.h
         ${MBGL_ROOT}/render-test/ios/iosTestRunner.mm
+        ${MBGL_ROOT}/render-test/ios/ZipArchive/ZipArchive.h
```

Comment:
Could we use some MIT / BSD project from github as a submodule?

Reply:
Discussed with @alexshalamov  and @tmpsantos,  ZipArchive will be added as a submodule and it will only be used for render tests.